ethereum/walletBackend.go

```diff
@@ -0,0 +1,60 @@
+package ethereum
+
+import (
+	"errors"
+	"os"
+
+	"github.com/ethereum/go-ethereum/accounts/keystore"
+	"github.com/ethereum/go-ethereum/common"
+
+	ethwallet "perun.network/go-perun/backend/ethereum/wallet"
+	"perun.network/go-perun/wallet"
+)
+
+// WalletBackend provides ethereum specific wallet backend functionality.
+type WalletBackend struct {
+	encParams scryptParams
+}
+
+const (
+	standardScryptN = keystore.StandardScryptN
+	standardScryptP = keystore.StandardScryptP
+	lightScryptN    = 2
+	lightScryptP    = 1
+)
+
+type scryptParams struct {
+	N, P int
+}
+
+// InitTestWalletBackend initializes an ethereum specific wallet backend with weak encryption parameters.
+// These wallets can be unlocked faster and intended for use in tests only.
+// For all other purposes, use the InitWalletBackend function instead.
+func InitTestWalletBackend() *WalletBackend {
```

Comment:
Please make all testing functions either private or put them in `test` packages.

Reply:
yes, see this [comment](https://github.com/direct-state-transfer/dst-go/pull/50#discussion_r444897627) above.

Reply:
Updated. I have reorganized the packages as stated in [previous comment](https://github.com/direct-state-transfer/dst-go/pull/50#discussion_r451661152). So it is moved to `etherem/test` package.